doc/release-process.md

```diff
@@ -144,7 +141,7 @@ Follow the relevant Guix README.md sections:
 pushd ./guix.sigs
 git add "${VERSION}/${SIGNER}"/noncodesigned.SHA256SUMS{,.asc}
 git commit -m "Add attestations by ${SIGNER} for ${VERSION} non-codesigned"
```

Comment:
Added, but had to use `-m` for macOS users.

Reply:
> I don't think macOS can run Guix?

Correct, but it could be run under virtualisation on a macOS system, and `NO_SIGN` used to extract the attestations for signing on the host. `--machine` also wouldn't work for anyone signing on a BSD machine, or similar, so `-m` seems reasonable.

Reply:
Right. I'm just going to remove this then. As it's impossible to provide generic instructions that work with all our supported signing methods.